### ISSUE TITLE
Build Fail in ReadtheDocs

### ISSUE THREAD

**Issue by josesanchez1815** [josesanchez1815/mkdocs-vidlet-hosting#1]
Sorry I need to be asking for help sooner. I'm trying to build the mkdocs site. But it doesn't appear as it does on the local render. Here are some photos of what I'm trying to do.

Here is an error I find on the readthedocs
![image](https://user-images.githubusercontent.com/62918474/84331827-2c6fc300-ab40-11ea-9281-c3906ef0f86f.png)

**Comment by josesanchez1815** [comment]
It should look like this...but it doesn't.
![image](https://user-images.githubusercontent.com/62918474/84332280-5ecdf000-ab41-11ea-8d9c-e30bbbf6174e.png)

@ctb @ACharbonneau @luizirber
@abhijna

**Comment by josesanchez1815** [comment]
@marisalim

**Comment by ACharbonneau** [comment]
Generally speaking, it's almost impossible to debug code from screenshots, please always try to post links to the code, and specific error messages when you're looking for help. 

The configuration error you're getting suggests that your directory structure is wrong. Probably you don't have a file in the place RTD thinks is the default, and you didn't change that setting to tell it where to look...but I can't really be more specific without more information. If you post a link to your repo, and the message from `raw` in RTD, I can be more helpful :)

**Comment by josesanchez1815** [comment]
Repo Link: https://github.com/josesanchez1815/mkdocs-vidlet-hosting

**Comment by ACharbonneau** [comment]
That's the error when you click `View raw`?

**Comment by josesanchez1815** [comment]
link view raw 
https://readthedocs.org/api/v2/build/11231712.txt

**Comment by ACharbonneau** [comment]
lol
I literally just realized this issue is IN the repo you're building from. I thought this was in the dib-lab issues

**Comment by ACharbonneau** [comment]
good news! I've replicated your error!

**Comment by ACharbonneau** [comment]
https://cfde-mkdocs-vidlet-hosting.readthedocs-hosted.com/en/latest/
https://github.com/ACharbonneau/mkdocs-vidlet-hosting

**Comment by ACharbonneau** [comment]
You have a folder where the only thing inside of it is another folder that actually holds your website stuff. So, your site directory was ending up two folders deep, which RTD assumes means it's in the docs directory, because 2 folders deep means docs. Or at least that's my best explanation without actually understanding how RTD is coded. 

TLDR; RTD is expecting a specific directory structure. Having the extra top level folder broke it. I took it out.

**Comment by josesanchez1815** [comment]
Yay! I did it! Thanks @ACharbonneau 
![image](https://user-images.githubusercontent.com/62918474/84419535-98e4d380-abcd-11ea-87af-8b1e8433683f.png)
